motivaa-backend/motivaa-app/src/test/java/backendtesting/CookieHandler.java

```diff
@@ -0,0 +1,60 @@
+package backendtesting;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import lombok.extern.log4j.Log4j2;
+
+import java.util.Arrays;
+
+@Log4j2
+public class CookieHandler {
+    public static String retrieveJSessionIdCookie(String ENVIRONMENT_HOST,
+                                                  String endpoint,
+                                                  Boolean shouldRequestBeLogged) {
+
+        Response initialRequestResponse = initiateInitialRequest(
+                ENVIRONMENT_HOST,
+                endpoint,
+                shouldRequestBeLogged
+        );
+        return extractCookieFromResponseHeader(initialRequestResponse);
+    }
+
+    private static Response initiateInitialRequest(
+            String ENVIRONMENT_HOST,
+            String endpoint,
+            Boolean logEnabled) {
+        if (logEnabled) {
+            return RestAssured.given()
```

Comment:
this is too much of a code duplication just to set the logging. I look into the RestAssured library a little bit, could you use maybe the RestAssured.config().logConfig() to specify this, or solve it in an other way?

Reply:
This would be good to solve because I can see that in RequestMaker you have duplicates too because of the logging.

Reply:
they also have this if you want to achieve this:
https://github.com/rest-assured/rest-assured/wiki/Usage#log-if-validation-fails

Reply:
You are right. Made it a lot better now. Done - thanks!